src/components/Settings/SettingsTabs.js

```diff
@@ -0,0 +1,39 @@
+// @flow
```

Comment:
This code isn't getting used, is it? I see another component called `SettingsTabs` in `components/Settings/Settings.js` that seems to be the one we're actually rendering.

Reply:
Removed in latest commit